core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -313,8 +315,9 @@ impl BlockBandwidthRequests {
     ProtocolSchema,
 )]
 pub struct BandwidthSchedulerState {
-    /// Random data for now
-    pub mock_data: [u8; 32],
+    pub link_allowances: Vec<((ShardId, ShardId), u64)>,
+    /// Sanity check hash to assert that all shards run bandwidth scheduler in the exact same way.
+    pub sanity_check_hash: CryptoHash,
```

Comment:
Can you also add what is being hashed here?

Reply:
Added a short comment